scripts/generate-markdown/markdown.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const fs = require('fs');
+const markdownMagic = require('markdown-magic'); // eslint-disable-line import/no-extraneous-dependencies
+
+const defaults = {
```

Comment:
Do we need these defaults or can we hard code them?

Reply:
I went ahead and hard-coded them. Some of it didn't really seem necessary for our use case.